taurus/demo/tests/test_sac.py

```diff
@@ -21,3 +22,6 @@ def test_sac():
     # Make sure there's no migration with repeated serialization
     for row in sac_tbl:
         assert je.dumps(je.loads(je.dumps(row))) == je.dumps(row)
+
+    # Verify that the serialization trick for mocking a structured table works
+    json.dumps(json.loads(je.dumps(sac_tbl))["object"], indent=2)
```

Comment:
What does it mean to verify that it works? Is it sufficient that this this object can be dumped without error or is there something else being tested?

Reply:
`verify that it works` literally means doesn't throw an exception.  Not great form, but we're about to obsolete this pathway to tables.